.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
This shouldn't be added - I'm thinking this needs to be rebased against develop, as I think the additions to .gitignore are already there on develop?

Reply:
I rebased a couple of times (though I'm still not sure I'm doing it correctly).
The changes to .gitignore are on my branch. I think the problem is that .vscode got merged into develop when I accepted Isaac's PR.
I will delete the .vscode directory from both the develop branch and my branch.